### PR TITLE
Better page xcp-ng

### PR DESCRIPTION
**Pull request by Marcpezin**
Here is a new landing page when using the index.

### CODE REVIEW

File: src/xenserver/opt/xensource/www/XCP-ng-index.html [hunk 1]
```diff
@@ -3,181 +3,253 @@
...
+			<a href="XCP-ng-Center.msi" class="btn">Download</a>
+		</li>
+		<li>
+			<i class="fab fa-chrome"></i>
+			<h4>Xen Orchestra Virtual Appliance</h4>
+			<p><a href="https://xen-orchestra.com/#!/xo-home">Xen Orchestra</a> is a virtual appliance covering all the administration, management, backup and cloud features you may need in your infrastructure.</p>
+			<br>
+			<br>
+			<div id="ex1" class="modal">
+				<p>SSH to your XenServer and execute the following:</p>
```
Comment (olivierlambert) [inline src/xenserver/opt/xensource/www/XCP-ng-index.html]: to your XCP-ng ;)

File: src/xenserver/opt/xensource/www/XCP-ng-index.html [hunk 2]
```diff
@@ -3,181 +3,253 @@
...
+				<p style="font-style: italic">This will automatically download/import/start the XOA appliance.</p>
+				<br>
+				<a href="#" rel="modal:close">Close</a>
+			</div>
+			<p><a href="#ex1" class ="btn" rel="modal:open">Quick deploy</a></p>
+		</li>
+		<li>
+			<i class="fab fa-osi"></i>
+			<h4>Xen Orchestra Community Edition</h4>
+			<p>You can build Xen Orchestra by yourself to unlock all the features. There is no pro support nor updater for this version. Recommended for homelab.</p>
```
Comment (olivierlambert) [inline src/xenserver/opt/xensource/www/XCP-ng-index.html]: Only recommended for home lab. Maybe emphasis on "no pro support"?

**Review by olivierlambert — CHANGES_REQUESTED** [review]

**Comment by Marcpezin** [comment]
> I though emphasis mean't _no pro support_ or **no pro support** in the existing "There is no pro support" :)

@stormi @stormi I think it's more useful in the end of the sentence like a big warning than lost in the short description. If you don't like it, I can change it tho - let me know :)

All the resources that were previously downloaded from other sources are now located in the asset (including Font Awesome). The image is now in JPG and optimized to be as light as possible. 

If you don't have more improvement to suggest, @Fohdeesha could make a quick reread and then we could merge it.

**Comment by stormi** [comment]
> > I though emphasis mean't _no pro support_ or **no pro support** in the existing "There is no pro support" :)
> 
> @stormi I think it's more useful in the end of the sentence like a big warning than lost in the short description. If you don't like it, I can change it tho - let me know :)

At the time of my comment "no pro support" was there twice. Now it's there only once so no objection.

> 
> All the resources that were previously downloaded from other sources are now located in the asset (including Font Awesome). The image is now in JPG and optimized to be as light as possible.
> 
> If you don't have more improvement to suggest, @Fohdeesha could make a quick reread and then we could merge it.

Wow, 1436 files (most of them just for Font Awesome) for a single webpage, this is a lot :)
Is there a way to reduce that by identifying the assets we really need?

Lastly, can you squash all your commits into one then force push the branch? I'd like us to keep a clean and meaningful git history. You can use `git rebase -i master`, then force-push your branch. If not accustomed to the process we can see together.

**Comment by Marcpezin** [comment]
@stormi I have difficulties to align correctly images when I'm not using Font Awesome so I gave up for now with this idea. 

However, I have reduced the FA files to the strict minimum files required for this page. With less stuff, it breaks.

**Comment by stormi** [comment]
@Marcpezin That's good for me. We just need the review from @Fohdeesha I suppose, then you can squash the commits and I'll merge the pull request.

**Pull request merged**